src/styles.scss

```diff
@@ -132,191 +103,6 @@ body {
     vertical-align: sub;
   }
 
-  $list-title-height: 36px;
-
-  .list-content-menu, .table-selection-top {
-    align-self: start;
-  }
-
-  .table-selection-top {
-    height: $list-title-height;
-    min-height: $list-title-height;
-  }
-
-  // Style which places menu icon in upper right corner of table cell
-  .list-content-menu {
-    display: grid;
-    grid-template-areas:
-      'hd mn'
-      'progress .'
-      'tags .'
-      'cn .';
-    grid-template-columns: auto 50px;
-    grid-template-rows: $list-title-height 1fr min-content min-content;
-
-    .header {
-      grid-area: hd;
-      margin: 8px 0;
-    }
-
-    .content {
-      grid-area: cn;
-      text-align: justify;
-      text-justify: inter-word;
-      max-height: 15rem;
-      overflow-y: auto;
-      margin: 0.5rem 0;
-    }
-
-    .menu {
-      grid-area: mn;
-    }
-
-    .tags-list {
-      grid-area: tags;
-    }
-
-    .course-progress {
-      grid-area: progress;
-      margin-bottom: 0.5rem;
-    }
-
-  }

```

Comment:
Removing this from the `styles.scss` results in a duplication in the `courses.scss`, `resources.scss` & `meetups.component.scss`